src/lib.rs

```diff
@@ -126,6 +130,7 @@ impl fmt::Display for Error {
     }
 }
 
+#[cfg(feature = "std")]
 impl std::error::Error for Error {}
```

Comment:
:(

Only need to wait two more years for `core::error::Error` at this pace of MSRV growth